src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -2872,21 +2871,31 @@ namespace WaterToAirHeatPumpSimple {
                          .glycol->getSpecificHeat(state, state.dataSize->PlantSizData(PltSizNum).ExitTemp, RoutineNameAlt);
 
                 if (simpleWatertoAirHP.WAHPType == WatertoAirHP::Heating) {
-                    RatedWaterVolFlowRateDes = simpleWatertoAirHP.RatedCapHeat / (state.dataSize->PlantSizData(PltSizNum).DeltaT * Cp * rho);
+                    RatedWaterVolFlowRateDes =
+                        (1 + 1 / RatedHeatCOP) * simpleWatertoAirHP.RatedCapHeat / (state.dataSize->PlantSizData(PltSizNum).DeltaT * Cp * rho);
+                    if (simpleWatertoAirHP.CompanionCoolingCoilNum > 0) {
+                        auto const &companionCoolingCoil =
+                            state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionCoolingCoilNum);
+                        RatedWaterVolFlowRateDes = max(RatedWaterVolFlowRateDes,
+                                                       (1 + 1 / RatedCoolCOP) * companionCoolingCoil.RatedCapCoolTotal /
+                                                           (state.dataSize->PlantSizData(PltSizNum).DeltaT * Cp * rho));
```

Comment:
Here we're dealing with the companion cooling coil so the evaporator is on the load side and the condenser is on the source side.

So if we start with `Qc = Qe + P` and because we're dealing with heating operation `COP = Qe / P` so `Qc = Source Side Load = Qe + P = Qe (1 + 1 / COP)` and `Qe = companionCoolingCoil.RatedCapCoolTotal`. So I think this is correct.

Then, we're taking the maximum of either the calculated heating coil evaporator flow rate or the cooling coil condenser flow rate.

Reply:
We must be cautious about which `COP` value is used to relate the source-side and load-side heat transfer rates. In your equation, `Qc = Qe + P = Qe (1 + 1/COP)`, the `COP` should be the cooling `COP` as shown below:

```
COP_c = Q_evap / P 
COP_h = Q_cond / P
```

So,

`Q_cond = Q_evap + P = Q_evap * (1 + 1/COP_c)`

Why do you have to take the maximum?

Reply:
Right, cooling coil COP for the cooling coil calculations and heating coil COP for heating coil calculations.

If we have companion coils in the system (both water-to-air heating and cooling coils), we could end-up with different calculated water flow rates. Since we need only one value for the system, I assumed that we would want to keep the larger of the two. Do you have a suggestion?

Reply:
I don't see a problem taking the maximum flow as long as the flow rates are determined based on their respective design loads, design delta T, and CP values appropriate for cooling and heating loops. In this case, it is how you arrive at the `Design Load` that leads to different flow rates.

Reply:
Also, the assumption here for the companion coil is that it has already been sized. You might want to check if RatedCapCoolTotal == DataSizing::AutoSize before this calculation?